### DISCUSSION TITLE
Blockquotes suddenly formatted with extra-large text

### DISCUSSION THREAD

**Discussion by krangor** [buttondown/discussions#40] [Q&A] [answered]
Hi! Me again—just noticed that in previewing both email and web versions, my blockquotes are suddenly blown-up and huge and no CSS fiddling seems to be able to stop it. Examples:
<img width="490" alt="image" src="https://github.com/user-attachments/assets/7efb98d4-7e04-41f9-b926-a160e0c153a7">
<img width="493" alt="image" src="https://github.com/user-attachments/assets/e6267c1c-7dd2-47d4-9e6e-cf614c5a5693">

Curiously, the archives seem fine. Here's how it should look:
<img width="635" alt="image" src="https://github.com/user-attachments/assets/a8f869e5-c266-4d75-9e95-4fc4e8afd65f">

As well, I keep getting this warning whenever I load an email:
<img width="621" alt="image" src="https://github.com/user-attachments/assets/74301462-6b8c-470d-bc8c-b82aeffa94d6">

**Comment by krangor** [comment]
Update: looking at the text in Markdown mode, I see this:
![image](https://github.com/user-attachments/assets/096add85-04df-4da0-bbca-1197d0d82ff7)
Removing "h1" seems to fix it, but I'm confused how it got there in the first place.

**Reply by jmduke** [reply]
Hmm — do you remember if these are archives you've imported, or o.g. to Buttondown? (Also mind sharing a URL so I can poke around?)

**Reply by krangor** [reply]
These are totally new! And it's happening in both the emails I sent out AND the archive page, weirdly enough.
Here's what I sent out Friday, blockquote text weirdly h1-ed in both formats.
https://www.markslutsky.com/archive/something-good-103-all-i-want-to-do-is-read/

**Reply by krangor** [reply]
OH... and I should say, I actually manually fixed this in the editor after it published, so you won't see the problem any longer.

**Reply by jmduke** [reply]
Random, but: did you copy and paste the quotes from somewhere else (maybe a different editor you were using, for instance?)

**Reply by krangor** [reply]
Ahh, now maybe we're on to something here. I did copy it from a webpage. Since there doesn't seem to be any way to style text within the blockquote in the editor (feature request btw!), it wouldn't have occurred to me that it might have brought some along. Let me try a test in a draft.

**Reply by krangor** [reply, marked as answer]
This was it! Including this screenshot to point out how blockquote styling is invisible (and seemingly immutable?) in the Fancy editor.

<img width="1168" alt="image" src="https://github.com/user-attachments/assets/96351065-9ac7-4acd-976e-1cf11173395d">

**Reply by jmduke** [reply]
Aha, excellent. And yes, will file that..
